packages/core/components/ExternalInput/styles.module.css

```diff
@@ -1,7 +1,3 @@
-.ExternalInput {
-  font-family: var(--puck-font-stack);
-}
-
```

Comment:
Does the ExternalInput need to be added to the tabIndex?

Reply:
I think all relevant ExternalInput controls should be tabindex-able already?

Reply:
Yes, sorry, Safari testing foible

Reply:
Are the rows supposed to be keyboard accessible?

No worries if not, just wanted to sanity check

Reply:
They need to be, but they're currently not. One of the issues to be spun out.  Suspect we may need to switch to an add button inside a cell, but this is one for later.

> a11y of clickable table rows in ExternalInput

Reply:
Best to wait until I'm down with the ExternalInput changes too, then!